interlay/interbtc-ui
# Rui/ltv

# Interbtc UI Pull Request Template

## Description

Adds LTV(loan to value) Meter as a new standard to represent how healthy user loans are.

Figma: https://www.figma.com/file/cTdXvqV3FdN5rpND3yEGxI/Interlend-v0.1?node-id=1368%3A3466&t=vP1YWrUuxYcPeLkk-4

## Current behaviour (updates)

Currently we show a Health Score

## New behaviour

New LTV meter that replaces old health score

## Reproducible testing steps:

- Open a lend position and make it as collateral
- Open a borrow position
- While opening a borrow position you should be presented with the new LTV Meter
- You need to be under the yellow indicator to be able to open a borrow position
- After openning a borrow positions, you should be able to see new LTV Meter in the Lending page
- This section should present you with your current total borrow amount as well as the collateral amount.
- You should be presented as well with a loan status indicator.
